composer.json

```diff
@@ -7,6 +7,9 @@
         "AFL-3.0"
     ],
     "type": "maho-source",
+    "repositories": [
+        { "type": "vcs", "url": "https://github.com/justinbeaty/maho-composer-plugin.git" }
+    ],
```

Comment:
```suggestion
```

I'd proceed removing this and publishing to the real one

Reply:
Sure, I just had one more change in the plugin for disable_local_modules support with the new PSR-0 support. I'll push a PR to the composer plugin now and we can review there.